### PR TITLE
Add logging repository requests

### PR DESCRIPTION
**Pull request by djyou**

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py [hunk 1]
```diff
@@ -25,6 +26,8 @@ def _obtain_data_from_registry(login_server, path, resultIndex, username, passwo
             )
         )
 
+        log_request(None, response.request)
```
Comment (troydai) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py]: Please correct me if I'm wrong. According to this: http://msrest.readthedocs.io/en/latest/_modules/msrest/http_logger.html. The `log_request` will log all the headers, including Authorization header. Won't this be a security concern?

Comment (SajayAntony) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py]: yes exactly. This is what the PR does. --debug already outputs the BEARER token as a part of the output. We need to see the authorization header that is being sent similar to the BEARER token to ARM.

Comment (troydai) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py]: ok, got it. so it is expected. i rest my case, sir.

**Comment by djyou** [comment]
Closed due to a security concern in logging basic authentication headers.

**Pull request closed without merging**